data-model/src/main/avro/LogEvent.avdl

```diff
@@ -0,0 +1,19 @@
+@namespace("org.hypertrace.core.datamodel")
+protocol LogEventProtocol {
+  import idl "Attributes.avdl";
+
+  record LogEvent {
+    long timestamp_nanos = 0;
+
+    /** tenant_id serves the purpose of identifying tenant **/
+    union { null, string } tenant_id = null;
```

Comment:
Isn't`tenant_id` expected?

Reply:
Keeping it nullable is just adding some flexibility around fields.